docs/source/development/style-guide/requirement-and-issue-tracking.rst

```diff
@@ -0,0 +1,120 @@
+.. _requirement-and-issue-tracking:
+
+Requirement and Issue Tracking
+------------------------------
+
+Definitions
+^^^^^^^^^^^
+=================== ===================================================================================================================================================================================================================================================
+Name                Definition
+=================== ===================================================================================================================================================================================================================================================
+Level 4 requirement requirements that have been decomposed to the SDC level. Level 4 requirements have already been defined.
+Level 5 requirement requirements decomposed from the level 4 requirements that better define how the SDC plans to satisfy the parent level 4 requirement, while not defining any implementation details. Example: Backup *Metadata* to AWS backup account vs. Backup *OpenSearch* to AWS backup account
+Implementation task Github issue that better defines how a certain feature will be implemented. Implementation tasks are tracked under the parent level 5 requirement as a sub-task. Each implementation task should be small enough to be handled in a single PR.
+SPIKE               Github issue created to investigate different architecture design approaches. Like implementation tasks, SPIKEs should be tracked under the applicable level 5 requirement as a sub-task.
+=================== ===================================================================================================================================================================================================================================================
+
+Requirements Overview
+^^^^^^^^^^^^^^^^^^^^^
+
+Requriements Mapping
+~~~~~~~~~~~~~~~~~~~~
+
+* Each level 4 requirement is mapped to at least one SIT.
+* Each level 4 requirement relevant to the upcoming SIT is decomposed into level 5 requirements that, together, cover the testing requirements of the SIT.
```

Comment:
```suggestion
* Each level 4 requirement within an upcoming SIT is split into level 5 requirements. These level 5 requirements, taken together, should completely cover the level 4 requirements. 
```

This bullet point was a little jargon-y and I had trouble understanding what you meant. Let me know if my edit is correct!

Reply:
I like that a lot better! The only thing I would change is "Each level 4 requirement being tested in the upcoming SIT" or "Each level 4 requirement mapped to the upcoming SIT". I just don't want to say "within" because level 4s can span multiple SITs. Do you have a preference?

Reply:
I think your change makes sense!